windows/bazel/get_workspace_status.bat

```diff
@@ -0,0 +1,3 @@
+@ECHO OFF
+bash -c "bazel/get_workspace_status %*"
```

Comment:
Same question again as earlier: which Bash do you expect to use here?
Maybe it'd advantageous to check that the right Bash is the first on the PATH.